cypress/integration/show_work_orders.spec.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+beforeEach(() => {
+  cy.getCookies().should('be.empty')
+  cy.setCookie('hackneyToken', Cypress.env('GSSO_TEST_KEY'))
+  cy.getCookie('hackneyToken').should(
+    'have.property',
+    'value',
+    Cypress.env('GSSO_TEST_KEY')
+  )
```

Comment:
can you please use the login function? as well as move this `beforeEach` block inside the `describe` block